scripts/db_migrator.py

```diff
@@ -899,6 +899,17 @@ def common_migration_ops(self):
         else:
             log.log_notice("Asic Type: {}, Hwsku: {}".format(self.asic_type, self.hwsku))
 
+        route_table = self.appDB.get_table("ROUTE_TABLE")
+        for route_prefix, route_attr in route_table.items():
+            if 'weight' not in route_attr:
+                if type(route_prefix) == tuple:
+                    # IPv6 route_prefix is returned from db as tuple
+                    route_key = "ROUTE_TABLE:" + ":".join(route_prefix)
+                else:
+                    # IPv4 route_prefix is returned from db as str
+                    route_key = "ROUTE_TABLE:{}".format(route_prefix)
+                self.appDB.set(self.appDB.APPL_DB, route_key, 'weight','')
```

Comment:
@prsunny please note that I have not initialized the values to `1.1.1.1`.  The failure was due to missing attributes, but not the value.

During reconciliation, fpmsyncd will still put the default (or the intended) value as the weight. Hardcoding the value to `1.1.1.1` can be thus avoided. Please let me know if you have any concerns.

Reply:
Isn't the default weight '1'?

Reply:
The default value is indeed `1` - for every nexthop.

So, for example, the value of `weight` for 4 nexthops will be `1,1,1,1`.

Hardcoding here will require the logic to check how many nexthops are present in the route and then generate the default value based on that.